src/Controller/Component/AuthComponent.php

```diff
@@ -630,32 +620,25 @@ public function logout()
         }
         $user = (array)$this->user();
         $this->dispatchEvent('Auth.logout', [$user]);
-        $this->session->delete($this->sessionKey);
         $this->session->delete('Auth.redirect');
```

Comment:
I think the goal of this features is to be able to avoid the the session entirely, so this line should probably be moved to the other storage as well.